ConnectorDynamo/ConnectorDynamoExtension/SpeckleExtension.cs

```diff
@@ -35,7 +35,10 @@ public void Loaded(ViewLoadedParams viewLoadedParams)
 
       Setup.Init(HostApplications.Dynamo.GetVersion(HostAppVersion.vRevit), HostApplications.Dynamo.Slug);
     }
-    catch (Exception e) { }
+    catch (Exception ex) when (!ex.IsFatal())
+    {
+      SpeckleLog.Logger.Error(ex, "Failed to load Speckle extension");
```

Comment:
Should this become a `Fatal`
Also, is this considered a "Top level" exception handler?
If so, would we want to apply the DUI2 strategy of a normal try catch here? (with no `ex.IsFatal()` check)?